include/ultra64/gbi.h

```diff
@@ -1430,120 +1774,155 @@ typedef struct {
  * } Gsegment;
  */
 typedef struct {
-        int     cmd:8;
-        int     pad0:8;
-        int     mw_index:8;
-        int     number:8;
-        int     pad1:8;
-        int     base:24;
+    int cmd      : 8;
+    int pad0     : 8;
+    int mw_index : 8;
+    int number   : 8;
+    int pad1     : 8;
+    int base     : 24;
 } Gsegment;
 
 typedef struct {
-        int     cmd:8;
-        int     pad0:8;
-        int     sft:8;
-        int     len:8;
-        unsigned int    data:32;
+    int          cmd  : 8;
+    int          pad0 : 8;
+    unsigned int sft  : 8;
+    unsigned int len  : 8;
+    unsigned int data : 32;
 } GsetothermodeL;
 
 typedef struct {
-        int     cmd:8;
-        int     pad0:8;
-        int     sft:8;
-        int     len:8;
-        unsigned int    data:32;
+    int          cmd  : 8;
+    int          pad0 : 8;
+    unsigned int sft  : 8;
+    unsigned int len  : 8;
+    unsigned int data : 32;
 } GsetothermodeH;
 
 typedef struct {
-        unsigned char   cmd;
-        unsigned char   lodscale;
-        unsigned char   tile;
-        unsigned char   on;
-        unsigned short  s;
-        unsigned short  t;
+    unsigned char  cmd;
+    unsigned char  lodscale;
+#ifdef F3DEX_GBI_2
+    unsigned char  pad   : 2;
+    unsigned char  level : 3;
+    unsigned char  tile  : 3;
+#else
+    unsigned char  tile;
+#endif
+    unsigned char  on;
+    unsigned short s;
+    unsigned short t;
 } Gtexture;
 
 typedef struct {
-        int     cmd:8;
-        int     pad:24;
-        Tri     line;
+    int           cmd : 8;
+#ifdef F3DEX_GBI_2
+    unsigned char v0;
+    unsigned char v1;
```

Comment:
for like somewhere else, this could be
```suggestion
    unsigned char v0_x2;
    unsigned char v1_x2;
```
but as you said being super precise about these fields isn't really worth it and not very doable anyway so whatever